cmd/autoheal/awx_job.go

```diff
@@ -96,12 +96,14 @@ func (h *Healer) launchAWXJob(
 	template *awx.JobTemplate,
 	action *autoheal.AWXJobAction,
 	rule *autoheal.HealingRule,
+	alert *alertmanager.Alert,
 ) error {
 	templateId := template.Id()
 	templateName := template.Name()
 	launchResource := connection.JobTemplates().Id(templateId).Launch()
 	response, err := launchResource.Post().
 		ExtraVars(action.ExtraVars).
+		ExtraVar("alerts", alert).
```

Comment:
If we are calling this `alerts` then it should be an array of alerts. The idea of that is that in the future we may want to have rules that are activated only when multiple alerts are fired, and then this would contain all those alerts. If it is going to be a single alert, which is fine as well, then I'd suggest to name it `alert`.